app/controllers/gems_adoptions_controller.rb

```diff
@@ -24,6 +24,8 @@ def update
     adoption_request = AdoptionRequest.find(params[:id])
     gems_adoption = adoption_request.gems_adoption
     if gems_adoption.update(status: 'closed') && GemOwnershipTransfer.create(old_user: current_user, new_user_id: adoption_request.user_id, ruby_gem_id: gems_adoption.ruby_gem_id)
+      NotificationMailer.email_adoption_request_accepted(adoption_request.user, adoption_request.gems_adoption.ruby_gem).deliver_now
```

Comment:
Can you move this to its own private method? Maybe even to its own object? These controllers are starting do get bloated.